cpp/src/mpi_reduce.cpp

```diff
@@ -0,0 +1,43 @@
+#include <mpi.h>
+#include <iostream>
+
+int main(int argc, char** argv) {
+    MPI_Init(NULL, NULL);
+
+    int world_rank;
+    MPI_Comm_rank(MPI_COMM_WORLD, &world_rank);
+
+    int world_size;
```

Comment:
Looks like unused?

Reply:
world_size was unused. Removed that part.